src/main/java/org/broadinstitute/consent/http/service/DacService.java

```diff
@@ -306,13 +306,13 @@ private boolean isAuthUserChairOrMember(AuthUser authUser) {
     /**
      * Filter data access requests by the DAC they are associated with.
      */
-    List<DataAccessRequest> filterDataAccessRequestsByDac(List<DataAccessRequest> documents, AuthUser authUser) {
-        if (isAuthUserAdmin(authUser)) {
+    List<DataAccessRequest> filterDataAccessRequestsByDac(List<DataAccessRequest> documents, User user) {
+        if (User.hasUserRole(user, UserRoles.ADMIN)) {
```

Comment:
Instead of a static, it should be a plain utility method on user, i.e. 
```suggestion
        if (user.hasUserRole(UserRoles.ADMIN)) {
```